spec/v1/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## v1.1
+
+-   Clarified that subjects are assumed to be immuatble and that it is
```

Comment:
```suggestion
-   Clarified that subjects are assumed to be immutable and that it is
```

Reply:
Well, that's embarrassing.  I need to find a better setup when working with GitHub (one that provides spell-check :p).

Fixed!